src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Package/build/TypeMaps.props

```diff
@@ -1,6 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Project>
   <ItemGroup>
-    <AdditionalFiles Include="$(MSBuildThisFileDirectory)\WebTypeReplacements.typemap" />
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)\WebTypeReplacements.typemap">
```

Comment:
Why have it show up? We don't want them to change this file as it will be in the nuget cache (it should be considered readonly). If people need to add their own, they can add a typemap themselves.

Reply:
Good point about it being read only. I liked the idea of the file being visible so that (a) there's less magic about what the analyzers are doing and (b) they can use it as an example, but I see what you're saying. I'll update this to add `<Visible>false</Visible>` instead of Link.